### PR TITLE
Check in linear percent indicator state is null if the size is null

### PR DESCRIPTION
**Pull request by victormanuelfrancodev**
**Description**: 
It checks if size is null before assigning a measurement value. Sometimes it might happen that size comes as null, which is why the error occurs on certain occasions

**Error**:
```
Non-fatal Exception: io.flutter.plugins.firebase.crashlytics.FlutterError: Null check operator used on a null value. Error thrown Instance of 'ErrorDescription'.
       at RenderBox.size(box.dart:2001)
       at Element.size(framework.dart:4175)
       at _LinearPercentIndicatorState.initState.<fn>.<fn>(linear_percent_indicator.dart:165)
       at State.setState(framework.dart:1109)
       at _LinearPercentIndicatorState.initState.<fn>(linear_percent_indicator.dart:164)
       at SchedulerBinding._invokeFrameCallback(binding.dart:1146)
       at SchedulerBinding.handleDrawFrame(binding.dart:1091)
       at SchedulerBinding._handleDrawFrame(binding.dart:997)
```

**Related**: 

https://github.com/diegoveloper/flutter_percent_indicator/issues/174#issue-1341148691

### CODE REVIEW

File: example/ios/Runner.xcodeproj/project.pbxproj
Comment (victormanuelfrancodev) [inline example/ios/Runner.xcodeproj/project.pbxproj:1]: xd no vi que te envié esto hahaha

File: lib/linear_percent_indicator.dart [hunk 1]
```diff
@@ -170,12 +170,11 @@ class _LinearPercentIndicatorState extends State<LinearPercentIndicator>
     WidgetsBinding.instance.addPostFrameCallback((_) {
       if (mounted) {
         setState(() {
-          _containerWidth = _containerKey.currentContext?.size?.width ?? 0.0;
-          _containerHeight = _containerKey.currentContext?.size?.height ?? 0.0;
+          _containerWidth = (_containerKey.currentContext?.size != null) ? _containerKey.currentContext!.size!.width : 0.0;
```
Comment (diegoveloper) [inline lib/linear_percent_indicator.dart:173]: can you revert these changes? I think this is not the root of the problem

Comment (victormanuelfrancodev) [reply lib/linear_percent_indicator.dart]: fixed in the last commit 🫡

**Review by diegoveloper — APPROVED** [review]
Thank you @victormanuelfrancodev
